xml/ssh_banners.xml

```diff
@@ -1129,17 +1141,18 @@
     <param pos="0" name="os.product" value="Tru64 Unix"/>
     <param pos="0" name="os.cpe23" value="cpe:/o:hp:tru64:-"/>
   </fingerprint>
-  <fingerprint pattern="^(?:SSH-(\d\.\d)-)?ROSSSH$">
+  <fingerprint pattern="^ROSSSH$">
     <description>MikroTik RouterOS sshd</description>
```

Comment:
Verified in a sample data set that Roy's changes are correct.  The raw banner that we get back is `SSH-2.0-ROSSSH` and the processed banner that Recog sees is `ROSSSH`

Reply:
doesn't this differ by implementation of Recog consumer? e.g. I think raw banners are used for recog matching in Nexpose and InsightVM

Reply:
This was the only banner in the entire `ssh_banners.xml` file that had an optional `SSH-x.x` prefix, so to keep this fingerprint in line with all the others, we decided to take it out.

Also the comment at the very top of `ssh_banners.xml` indicates that these fingerprints should be for the part of the banner after that prefix, so it looked ok to take this out.

https://github.com/rapid7/recog/blob/master/xml/ssh_banners.xml#L3-L7

```
  <!--
  SSH "software revision and comment" strings (official RFC nomenclature for the part of
  the identification string after "SSH-x.x-") are matched against these patterns to
  fingerprint SSH servers.
  -->
```